app/routes/user/email/index.tsx

```diff
@@ -70,7 +86,55 @@ export default function () {
         </a>
         .
       </p>
-      {data && (
+      <h2>GCN Circulars</h2>
+      <p>
+        Subscriptions to <strong>Circulars</strong> are sent from GCN Circulars{' '}
+        {`<no-reply@${hostname}>`} and will be delivered to the email associated
+        with this account.
+      </p>
+      {userIsSubscribedToCircularEmails ? (
+        <>
+          <p>
+            You are currently subscribed to receive GCN Circulars via Email.
+          </p>
+          <Form method="post">
+            <input type="hidden" name="intent" value={'unsubscribe'} />
+            <Button type={'submit'} unstyled>
+              Unsubscribe
+            </Button>
+          </Form>
+        </>
+      ) : (
+        <>
+          <p>
+            You are not currently subscribed to receive GCN Circulars via Email.
+          </p>
+          <Form method="post">
+            <input type="hidden" name="intent" value={'subscribe'} />
+            <Button type={'submit'} unstyled>
+              Subscribe
+            </Button>
+          </Form>
+        </>
+      )}
```

Comment:
We can do better than this for the UX. I propose:

- [ ] A [segmented button group](https://trussworks.github.io/react-uswds/?path=/story/components-button-groups--segmented) with buttons `On` and `Off`, mimicking a toggle button.
- [ ] A spinner to the right of the button group that displays while the form is submitting (we have several examples of these in our code now)
- [ ] A green checkmark and the text `Saved` that displays to the right of the button group when the form submission has completed (we also have several examples of these)

@dakota002, is your Remix-fu up to the task?

Reply:
Challenge accepted!

Reply:
🥋 💪

Reply:
![image](https://user-images.githubusercontent.com/11023698/227611291-65aa8088-6761-499d-9417-49ef67dea71b.png)

![image](https://user-images.githubusercontent.com/11023698/227611354-c438fada-41d0-4509-8778-06f27f53f9a2.png)

![image](https://user-images.githubusercontent.com/11023698/227611417-abce5c87-58e7-45b4-985e-b8e541ed4314.png)